Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/InstanceSerializer.cpp

```diff
@@ -23,6 +23,53 @@ namespace AzToolsFramework
     {
         AZ_CLASS_ALLOCATOR_IMPL(JsonInstanceSerializer, AZ::SystemAllocator, 0);
 
+        namespace Internal
+        {
+            inline static const char* PathStartingWithEntities = "/Entities/";
+            inline static const char* PathMatchingContainerEntity = "/ContainerEntity";
+
+            //! Identifies the instance members to reload by parsing through the patches provided.
+            static void IdentifyInstanceMembersToReload(
+                PrefabDom& patches, AZStd::unordered_set<EntityAlias>& entitiesToReload, bool& shouldReloadContainerEntity)
+            {
+                for (const PrefabDomValue& patchEntry : patches.GetArray())
+                {
+                    PrefabDomValue::ConstMemberIterator patchEntryIterator = patchEntry.FindMember("path");
+                    if (patchEntryIterator != patchEntry.MemberEnd())
+                    {
+                        AZStd::string patchPath = patchEntryIterator->value.GetString();
+                        if (patchPath.starts_with(PathStartingWithEntities))
+                        {
+                            patchPath.erase(0, strlen(PathStartingWithEntities));
+                            AZStd::size_t pathSepartorIndex = patchPath.find('/');
+                            if (pathSepartorIndex != AZStd::string::npos)
+                            {
+                                AZStd::string entityAlias = patchPath.substr(0, pathSepartorIndex);
+                                entitiesToReload.emplace(AZStd::move(entityAlias));
```

Comment:
Slight performance improvement is to make the entityAlias an AZStd::string_view and pass that to the `unordered_set::emplace` function.

That reduces the operation to a single construction of an AZStd::string.
The current operation is a construction of an AZStd::string, followed by a move which is cheap, but can avoided.
```suggestion
                                entitiesToReload.emplace(patchPath.substr(0, pathSeparatorIndex));
```